rxbinding-kotlin/src/main/kotlin/com/jakewharton/rxbinding/view/RxView.kt

```diff
@@ -3,13 +3,37 @@ package com.jakewharton.rxbinding.view
 import android.view.DragEvent
 import android.view.MotionEvent
 import android.view.View
-import rx.Observable
 import com.jakewharton.rxbinding.internal.Functions
+import rx.Observable
 import rx.functions.Action1
 import rx.functions.Func0
 import rx.functions.Func1
 
 /**
+ * Create an observable of timestamps for `view`'s attaches.
+ * 
+ * *Warning:* The created observable keeps a strong reference to `view`. Unsubscribe
+ * to free this reference.
+ */
+public inline fun View.attaches(): Observable<Any> = RxView.attaches(this)
```

Comment:
I'm sad we can't make this `Unit` in Kotlin. Maybe we should write all the code in Kotlin and generate the Java bindings instead...

Reply:
Hindsight 🤔